code/qcommon/q_platform.h

```diff
@@ -97,10 +97,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define ID_INLINE __inline
 #define PATH_SEP '\\'
 
-#if defined( __WIN64__ ) 
+#if defined(__WIN64__) || defined(_WIN64)
```

Comment:
To allow the _M_ALPHA check after this to work, maybe check arch defines similar to Win32 section `#if defined(_M_X64) || defined(__x86_64__)` (untested, based on https://sourceforge.net/p/predef/wiki/Architectures/),